docs/ci-operations-group.md

```diff
@@ -0,0 +1,140 @@
+# KubeVirt CI Operations Group
+
+
+## Goal of this document
+
+The current group consists only of a couple of people who mostly are not only handling CI but need to take other tasks as well. This document is intended to make clear what the primary responsibilities are in order to avoid overloading the members of this group with tasks.
+
+
+## Primary members
+
+<table>
+  <tr>
+   <td></td>
+   <td></td>
+   <td></td>
+   <td colspan="5" ><b>Responsibilities / Area of interest</b></td>
+  </tr>
+  <tr>
+   <td><b>Human</b></td>
+   <td><b>Availability</b></td>
+   <td><b>TZ</b></td>
+   <td><b>BM</b></td>
+   <td><b>Operations</b></td>
+   <td><b>Prow</b></td>
+   <td><b>kubevirtci</b></td>
+   <td><b>SIG Support</b></td>
+  </tr>
+  <tr>
+   <td><a href="https://github.com/brianmcarey">Brian Carey</a></td>
+   <td>Mon-Fri</td>
+   <td>GMT+1</td>
+   <td>Primary</td>
+   <td>Primary</td>
+   <td>Primary</td>
+   <td>Backup</td>
+   <td>Backup</td>
+  </tr>
+  <tr>
+   <td><a href="https://github.com/dhiller">Daniel Hiller</a></td>
+   <td>Mon-Fri</td>
+   <td>GMT+2</td>
+   <td>Backup</td>
+   <td>Secondary</td>
+   <td>Primary</td>
+   <td>Backup</td>
+   <td>Primary</td>
+  </tr>
+  <tr>
+   <td><a href="https://github.com/enp0s3">Igor Bezukh</a></td>
+   <td>Sun-Thu</td>
+   <td>GMT+3</td>
+   <td></td>
+   <td>Backup</td>
+   <td>Secondary</td>
+   <td>Backup</td>
+   <td>Secondary</td>
+  </tr>
+  <tr>
+   <td><a href="https://github.com/xpivarc">Luboslav Pivarc</a></td>
+   <td>Mon-Fri</td>
+   <td>GMT+2</td>
+   <td></td>
+   <td>Backup</td>
+   <td>Secondary</td>
+   <td>Backup</td>
+   <td>Secondary</td>
+  </tr>
+</table>
+
+
+**Note: kubevirtci should be driven by those who use it: Developers. The SIGs.**
+
+### Contact channels
+
+* [#kubevirt-dev (Kubernetes Slack)](https://kubernetes.slack.com/archives/C0163DT0R8X)
+* [~~#kubevirt-ci (CoreOS Slack)~~](https://coreos.slack.com/archives/C01EX3K1FGE) - Note: ⚠️ DEPRECATED as it's RH internal
```

Comment:
Do we need to include this? I imagine this is only relevant for RH ppl who have not seen the deprecation notice.

Reply:
I included it for completeness, but yes, it makes sense to just remove it.